content/blog/news/CloudPlugin-GSOC'20.md

```diff
@@ -0,0 +1,139 @@
+    remove during the process of scaling down
+    
+    * [delete_cloud_group()](https://github.com/AcademySoftwareFoundation/OpenCue/blob/d4a2eca93d271dacb13ca2aaba9b531642c1dc6b/pycue/opencue/cloud/api.py#L58):
+    Deletion of the cloud group from the cloud provider
+    
+    `Google Cloud Platform` implementation of this abstract class can be found here: [GoogleCloudGroup](https://github.com/AcademySoftwareFoundation/OpenCue/blob/d4a2eca93d271dacb13ca2aaba9b531642c1dc6b/pycue/opencue/cloud/gce_api.py#L22)
+ 
+#### Plugin widget design
+
+The Cloud Manager plugin is a plugin for CueCommander. It implements the `AbstractTreeWidget` class 
+and registers itself as a CueCommander plugin. 
+
+The interface uses the main `CloudManager` api mentioned above, as the entry point to get all cloud groups
+associated with all the registered cloud providers
+
+
+## Cloud Plugin Setup
+
+#### Google Compute Engine setup
+
+Listing the minimum requirements for the cloud plugin to work with GCE
+
+{{% alert title="Note" color="info"%}}My deployment included setting up Cuebot on GCE{{% /alert %}}
+
+1. GCP project with Compute Engine API and Cloud Logging API enabled
+
+2. Setup `Application Default Credentials` and make sure the google python api client (which is a requirement for this plugin)
+ has access to it
+
+3. Use the `cloud_plugin_resources.yaml` under the `cuegui/config` directory to setup your GCP project name and
+the zone
+
+4. At least one instance template setup on the cloud: I was using a RQD Blender docker image uploaded as an
+[Instance Template](https://cloud.google.com/compute/docs/instance-templates) with its appropriate VM type
+
+
+#### Cloud Plugin Interface
+
+The Cloud plugin should be available to launch from the menu bar by navigating: 
+**Views/Plugins -> CueCommander -> Cloud Plugin**
+
+1. The cloud plugin interface consists mainly of two components:
+    1. Main window which shows the list of cloud groups available with all registered providers
+    2. A cloud group addition dialog that can be used to create a cloud group from the interface itself
+    
+#### Cloud Plugin functionality
+ 
+1. To start create a cloud pool group, use the `Add Cloud Group` button to launch the 
+dialog that will allow you to create a cloud group
+
+![Add Cloud Group Dialog](/blog/images/AddCloudGroupDialog.png)
+
+2. If the API request is successful you should now see an entry in the Cloud Plugin's main interface
+
+![After cloud group addition](/blog/images/MainInterfacePostAddition.png)
+
+3. The right-click content menu has two additional functionality: 
+    1. Resizing (scaling up/down) the number of instances in a group
+    2. Deleting the group altogether
+
+![Cloud Plugin Context Menu](/blog/images/CloudPluginContextMenu.png)
+
+
+## Further work and updates
+
+An Azure plugin that uses the same interface is currently a work in progress
```

Comment:
Let's get a Draft PR open for that work ASAP, even if it's not quite ready, so we can link to the PR here.

Reply:
@bcipriano  Addressed all the comments. 
I have created a draft PR for the azure plugin as well on the main repo and linked it here. Let me know if the updated document looks good.